metron-platform/metron-management/README.md

```diff
@@ -150,25 +150,165 @@ The functions are split roughly into a few sections:
 
 ### Parser Functions
 
-* `PARSER_STELLAR_TRANSFORM_ADD`
+#### `PARSER_CONFIG`
```

Comment:
All other functions in the readme are bullets.  I think I prefer this approach since it makes it easier for us to link directly to functions, but it's more important that we're consistent - either by changing this to bullets or the others to `####`.

Reply:
Thanks for fixing those up with your PR.